gcsfs/core.py

```diff
@@ -34,6 +38,7 @@
          "publicReadWrite"}
 DEFAULT_PROJECT = os.environ.get('GCSFS_DEFAULT_PROJECT', '')
 DEBUG = False
+RETRY_COUNT = 4
```

Comment:
One final request: could this be made a class attribute? I don't expect people to change it, but it seems to fit more obviously there.

Reply:
Done! Actually while doing that I noticed there's already a `retries`, so I switched to using that instead - it's easier to configure the class if there's a single number for retry attempts. I also switched a "+1" to "-1" to match the style of the other retry block, to keep things easier to maintain.